Multisig/UI/ClaimToken/ClaimSplashViewController.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  ClaimSplashViewController.swift
+//  Multisig
+//
+//  Created by Dmitry Bespalov on 05.09.22.
+//  Copyright © 2022 Gnosis Ltd. All rights reserved.
```

Comment:
not critical, but we should adjust the copyright

Reply:
@elgatovital this would have to be everywhere in the project, I'd suggest we do that at a later point